dts/bindings/i2c/snps,designware-i2c.yaml

```diff
@@ -10,3 +10,9 @@ include: [i2c-controller.yaml, pinctrl-device.yaml, pcie-device.yaml]
 properties:
   interrupts:
     required: true
+
+  aux_reg:
+    description: |
+      This value is used for auxiliary register access. For
+      other platform, this value should be default 0.
+    type: boolean
```

Comment:
We don't support per-instance aux interface selection in this driver (which is fine as we don't need it for emsdp or other target right now) - so probably we shouldn't use DT property for it - as in device tree we still can specify `aux_reg` for some DW I2C node and don't specify `aux_reg` for another - so it's may me misliging for end user.

We can have one Kconfig property instead (like we used to have in DW SPI driver before recent rewrite) - so it would be clear that we either AUX access enabled for all instances or for none of them.